c2corg_ui/static/js/user.js

```diff
@@ -113,6 +113,7 @@ app.UserController.prototype.errorLogout_ = function(response) {
   alert('logout error');
   console.log('logout error');
   console.log(response);
+  this.auth.removeUserData();
```

Comment:
Yes, I think it is important for the user experience and the security that the token be always removed from the browser when the user asks for a logout.